### PR TITLE
[Security] Expression voter

### PR DESCRIPTION
**Pull request by alavieille**
Adding missing documentation of expression voter

### CODE REVIEW

**Review by OskarStark — APPROVED** [review]
Nice one

File: components/security/authorization.rst [hunk 1]
```diff
@@ -159,6 +159,40 @@ role::
 
     $roleHierarchyVoter = new RoleHierarchyVoter($roleHierarchy);
 
+ExpressionVoter
+~~~~~~~~~~~~~~~~~~
+The :class:`Symfony\\Component\\Security\\Core\\Authorization\\Voter\\ExpressionVoter`
+supports :class:`Symfony\\Component\\ExpressionLanguage\\Expression` attributes
+and grants access based on the evaluation of expression (See :doc:`/security/expressions` )
```
Comment (OskarStark) [inline components/security/authorization.rst]: s/(See/(See/

File: components/security/authorization.rst [hunk 2]
```diff
@@ -159,6 +159,40 @@ role::
 
     $roleHierarchyVoter = new RoleHierarchyVoter($roleHierarchy);
 
+ExpressionVoter
+~~~~~~~~~~~~~~~~~~
```
Comment (OskarStark) [inline components/security/authorization.rst]: This underline is longer than the headline

**Comment by OskarStark** [comment]
@wouterj I would like to have a review here

File: components/security/authorization.rst (hunk 1 above)
Comment (alavieille) [reply components/security/authorization.rst]: Fixed, thank

File: components/security/authorization.rst (hunk 2 above)
Comment (alavieille) [reply components/security/authorization.rst]: Fixed.

File: components/security/authorization.rst [hunk 3]
```diff
@@ -157,6 +157,41 @@ role::
...
+    $token = ...;
+
+    // any object
+    $object = ...;
+
+    $expression = new Expression(
+        '"ROLE_ADMIN" in roles or (not is_anonymous() and user.isSuperAdmin())'
+    )
+
+    $vote = $expressionVoter->vote($token, $object, array($expression));
```
Comment (noniagriconomie) [inline components/security/authorization.rst]: Replace array by []

Comment (noniagriconomie) [reply components/security/authorization.rst]: This should be merged
It is really a component worth knowing :)

**Comment by javiereguiluz** [comment]
@alavieille thanks a lot for contributing the docs for this missing feature! We merged it in the 3.4 branch so it's available for all Symfony Doc versions. Thanks!

**Pull request closed without merging**